### DISCUSSION TITLE
713

### DISCUSSION THREAD

**Discussion by giscus** [jojoldu/blog-comments#894] [Comments]
# 713

언니네이발관이 결성된 계기를 참 좋아한다. (이미지 출처: 나무위키) 아래 이야기는 루리웹에서 가져온 것이지만, 대부분의 커뮤니티에서 이미 공개되어 있는 이야기이다. PC통신시절 하이텔 메탈 음악 동호회에는 mypsb라는 전설의 악플러가 있었다. 지 맘에 안드는 뮤지션은 이 놈 저 놈 가릴것없이 닥치는대로 까내려서 악명이 자자했다. 당시 하이텔 동호회에는 현역 뮤지션들이 상당히 많았는데 음악평론활동(혐오물타기)을 하며 이들에게 꿀리기 싫었던 mypsb는 자신이 록밴드의 리더라고 입을 털고 다녔다. 물론 실제로는 밴드는 개뿔 악기 하나 다룰 줄 모르는 문외한이었다. 거짓말에 너무 심취한 나머지 현실과 망상의 경계가 모호해지는 수준에 이른 mypsb는 급기야 KBS라디오 '전영혁의 음악세계'라는 프로그램에 ..

https://jojoldu.tistory.com/713

**Comment by Rok93** [comment]
> 그 일을 진행하면서 그런 자격을 갖춘 사람이 되자.

정말 많이 공감되는  말이네요.
"함께 할 자격을 갖추고 함께 하겠다"는 생각으로 미루게 되면 결국 그 목표가 먼저 잊히거나 혹은 그 목표를 달성하는 데 걸리는 시간이 비약적으로 늘었던 반면 "에라 모르겠다 일단 해보자~"라는 생각으로 시도했던 것들로부터 더 긍정적인 결과, 경험을 많이 얻었던 것 같아요. 😊

너무 앞뒤 재지 말고 시도해 보는 뚝심이 필요하다고 생각합니다. 

오늘도 좋은 글 감사합니다. 👍

**Comment by AppleCEO** [comment]
그래서 동네 형의 오디션을 응원하러 왔던 동생이 보컬이 되고 하는 경우가 많았나봐요 ㅎㅎ 유명한 가수도 사실은 아는 형의 오디션에 와서 노래 불렀다가 뽑혔다고 하던데... ㄷㄷ

**Comment by JangJaehoon** [comment]
시도해 보는게 일단 미덕이군요!

**Comment by hajubal** [comment]
"자리가 사람을 만든다"
